src/forms.css

```diff
@@ -262,13 +266,43 @@ because it seems to be more cross-browser consistent with all HTML5 input types
 .select--s {
   font-size: var(--font-size-s);
   line-height: var(--line-height-s);
-  padding: 3px 24px 3px 6px; /* plus arrow */
+  padding: 3px 24px 3px 0; /* plus arrow */
 }
 
 .select--s + .select-arrow {
   right: 8px;
 }
 
+/**
+ * Modify a select component so that it has a border.
+ * Set the border (and arrow) color by adding a `select--{color}` modifier to the `select` element along with the `select--stroke` modifier (`*-dark` colors are not available).
```

Comment:
@adrianababakanian this documentation should clarify that the select--stroke modifier changes how the color modifier gets interpreted.